iceoryx_posh/include/iceoryx_posh/runtime/service_discovery.hpp

```diff
@@ -68,15 +70,17 @@ class ServiceDiscovery
     void invalidateTrigger(const uint64_t uniqueTriggerId);
     iox::popo::WaitSetIsConditionSatisfiedCallback
     getCallbackForIsStateConditionSatisfied(const popo::SubscriberState state);
-    roudi::ServiceRegistry m_serviceRegistry;
+
+    //  use dynamic memory to reduce stack usage,
+    /// @todo improve solution to avoid stack usage without using dynamic memory
```

Comment:
Please add a issue number here `/// @todo #123` and create the respective issue to clean this up.

Reply:
Yes sure, was a placeholder.